docs/src/using-parsers/7-abi-versions.md

```diff
@@ -14,7 +15,13 @@ A given version of the tree-sitter library is only able to load parsers which ha
 | >=0.20.3, <=0.24    | 13                     | 14                     |
 | >=0.25              | 13                     | 15                     |
 
-By default, the tree-sitter CLI will generate parsers using the latest available ABI for that version. Grammar authors can specify an older ABI (within the constraints _of the CLI_, which may be stricter than the library!) via the `--abi` option to the `generate` command:
+By default, the tree-sitter CLI will generate parsers using the latest available ABI for that version. Grammar authors can
+specify an older ABI (within the constraints _of the CLI_, which may be stricter than the library!) via the `--abi` option
+to the `generate` command:
+
 ```
 tree-sitter generate --abi=<DESIRED-ABI>
 ```
```

Comment:
I see that the ABI constraints for generating are already mentioned at https://tree-sitter.github.io/tree-sitter/cli/generate.html#--abi-version, so I think it's enough to link to this section from here and remove the parenthetical remark.